test/accountSpec.js

```diff
@@ -689,6 +686,88 @@ describe('Accounts', function () {
       })
     })
 
+    it('should resend notifications when asked to', function * () {
+      const listener = sinon.spy()
+      this.socket.on('message', (msg) => listener(JSON.parse(msg)))
+
+      const transfer = this.transfer
+
+      yield this.request()
+        .put(transfer.id)
+        .auth('alice', 'alice')
+        .send(transfer)
+        .expect(201)
+        .expect(validator.validateTransfer)
+        .end()
+
+      // TODO: Is there a more elegant way?
+      yield timingHelper.sleep(50)
+
+      const transferId = transfer.id.match('http://localhost/transfers/(.+)')[1]
+      this.socket.send(JSON.stringify({ type: 'request_notification',
+                                        id: transferId }))
+
+      yield timingHelper.sleep(50)
+
+      sinon.assert.calledTwice(listener)
```

Comment:
This is called twice because the first is when the transfer is put, and the second is because we explicitly request a notification via the ws connection? If that's the case, it seems like this could be two tests. 1. Check notification is sent when transfer is PUT to ledger. 2. Check notification is sent when we request one for a transfer already in the db.